geopandas/accessor.py

```diff
@@ -0,0 +1,91 @@
+from textwrap import dedent
+
+from pandas.core.accessor import _register_accessor
```

Comment:
@jorisvandenbossche is it okay to depend here on pandas internals? I don't want to reimplement the whole machinery in here but this can cause trouble potentially.